bin/check_tron_jobs.py

```diff
@@ -231,15 +231,22 @@ def main():
     cmd_utils.load_config(args)
     client = Client(args.server)
 
+    error_code = 0
     if args.job is None:
         jobs = client.jobs(include_job_runs=True)
         for job in jobs:
-            check_job_result(job=job, client=client, dry_run=args.dry_run)
+            try:
+                check_job_result(job=job, client=client, dry_run=args.dry_run)
+            except Exception:
+                log.info("check job result fails for job {}".format(job))
+                error_code = 1
     else:
         job_url = client.get_url(args.job)
         job = client.job_runs(job_url)
         check_job_result(job=job, client=client, dry_run=args.dry_run)
 
+    return error_code
+
 
 if __name__ == '__main__':
     main()
```

Comment:
sys.exit(main())

if you want to actually return the return code from the process